### ISSUE TITLE
Runtime issue: Axe is already running. Use `await axe.run()` to wait for the previous run to finish before starting a new run.

### ISSUE THREAD

**Issue by echezuria13** [dequelabs/axe-core#3528]
<< Please describe how you encountered this issue >>

axe-core version: `4.4.2`
axe-extension version: `4.29.1`

```
Error: Axe is already running. Use `await axe.run()` to wait for the previous run to finish before starting a new run.
    at it (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/axe-versions/latest/axe.js:12:78706)
    at axe.run (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/axe-versions/latest/axe.js:12:356294)
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/content.bundle.js:1:13565
```

**Comment by WilcoFiers** [comment]
Thank you for reporting. Can you provide some detail on how you got this error?

**Comment by WilcoFiers** [comment]
This issue's a duplicate of #3426. If you have any way to reproduce this problem, please provide details there. I'm going to close this, since we don't need two issues on the same topic.
